templates/parachain/runtime/src/genesis_config_presets.rs

```diff
@@ -0,0 +1,163 @@
+use cumulus_primitives_core::ParaId;
+
+pub use sp_consensus_aura::sr25519::AuthorityId as AuraId;
+
+use crate::{AccountId, SessionKeys, Signature, EXISTENTIAL_DEPOSIT};
+use alloc::{format, vec, vec::Vec};
+use serde_json::Value;
+use sp_core::{sr25519, Pair, Public};
+use sp_runtime::traits::{IdentifyAccount, Verify};
+
+/// Preset configuration name for a local testnet environment.
+pub const PRESET_LOCAL_TESTNET: &str = "local_testnet";
+
+/// Preset configuration name for a development environment.
+pub const PRESET_DEVELOPMENT: &str = "development";
```

Comment:
So in my PR, I moved this ot `sp-genesis-builder` and make it a 'standard' if I may say that. wdyt?

Reply:
Makes sense, I'll move it.